linera-core/src/worker.rs

```diff
@@ -1061,6 +1073,9 @@ where
                 value_hash: blobs[0].hash(),
             }
         );
+        NUM_ROUNDS_IN_PROPOSAL
+            .with_label_values(&[])
+            .observe(certificate.round.0 as f64);
```

Comment:
This would count the round numbers of all certificates (validated block, confirmed block or leader timeout) sent to us, regardless of whether they are valid, fully signed, and whether we already had them.

It does not count the rounds in block proposals (that would be `handle_block_proposal`).

Either way I wonder if it would be more useful to count the rounds in _valid_ certificates (and don't count it twice if we already have it) and the rounds in _valid_ block proposals.

Reply:
Oh, I see. Maybe I can rename this to `NUM_ROUNDS_IN_CERTIFICATE`, and add a label to capture the value of `CertificateValue`?
Then I can also add logs in `handle_block_proposal` in a following PR.
I'll do it but lmk what you think and I can alter it